docs/universal-gateway/domains/what-are-managed-domains.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: What Are Managed Domains?
+description: Learn about the free and paid domains that ngrok provides.
+---
+
+If you don't [already own a Domain](/universal-gateway/how-do-i-use-my-own-domain) that you want to use with ngrok, you can create a domain that is a subdomain of an ngrok-managed base domain.
+
+| Domain             | Availability                                      | HSTS Global Preload |
+| ------------------ | ------------------------------------------------- | ------------------- |
+| `ngrok.app`        | [Paid accounts](https://ngrok.com/pricing)        | Yes                 |
+| `ngrok.dev`        | [Paid accounts](https://ngrok.com/pricing)        | Yes                 |
+| `ngrok.pizza`      | [Paid accounts](https://ngrok.com/pricing)        | No                  |
+| `ngrok-free.app`   | Free accounts                                     | Yes                 |
+| `ngrok-free.dev`   | Free accounts                                     | Yes                 |
+| `ngrok-free.pizza` | Free accounts (coming soon)                       | No                  |
+| `ngrok.io`         | Discontinued and only available to older accounts | No                  |
```

Comment:
afaik, `ngrok.io` domains are still available on old and new paid accounts

Reply:
Resolved here:
- https://ngrok.slack.com/archives/C03LRGNSG6A/p1742571554464349